src/puppetlabs/puppetdb/query_eng/engine.clj

```diff
@@ -1546,19 +1609,19 @@
              :state state}
             :else {:node node :state state}))
 
-(pls/defn-validated op-to-string
-  "Convert an operator to a lowercase string or vector of such."
-  [op]
-  (if (vector? op)
-    (mapv string/lower-case op)
```

Comment:
Removed this because I couldn't find anywhere it was used (it would be used for something like `[["not=", "or", "and"], "certname", "foo"]`....)